runbooks-placement-strategy/README.md

```diff
@@ -0,0 +1,96 @@
+---
+creation_date: 2025-06-12
+issues:
+- https://github.com/giantswarm/roadmap/issues/2838
+owners:
+- https://github.com/orgs/giantswarm/teams/team-planeteers
+state: review
+summary: This RFC addresses the strategic placement of operational runbooks (currently called "Ops recipes") to improve accessibility, maintainability, and security while ensuring consistent structure and content sanitization.
+---
+
+# Strategic placement and standardization of operational runbooks
+
+## Problem statement
+
+Our operational runbooks (currently referred to as "Ops recipes") are currently scattered across multiple locations, specifically between our private intranet and public handbook. This fragmented approach creates several challenges:
+
+1. **Inconsistent accessibility**: Some runbooks are private when they could be public, limiting knowledge sharing and community contribution
+2. **Lack of standardization**: Runbooks follow different formats and structures across locations
+3. **Security concerns**: Some runbooks may contain sensitive information that needs to be identified and sanitized
+4. **Discovery issues**: Engineers struggle to find the right runbook when they're distributed across different platforms
+5. **Maintenance overhead**: Managing runbooks in multiple locations increases maintenance burden
+
+We need a unified decision for runbook placement that ensures:
+
+- Proper security and data sanitization
+- Consistent structure and format
+- Optimal accessibility for both internal teams and the community
+- Clear governance and maintenance processes
+
+### Decision maker
+
+Planeteers team.
+
+### Deadline
+
+30.07.2025
+
+### Who is affected / stakeholders
+
+- Platform Engineers (primary users of runbooks)
+- External community members (for public runbooks)
+
+### Preferred solution
+
+**Consolidate all runbooks in the public handbook with a (dedicated) runbooks section, following a standardized structure.**
+
+#### Key components:
+
+1. **Primary location**: Move all runbooks to a dedicated section in the public handbook (`docs/runbooks/`)
+2. **Content sanitization process**:
+   - In order to move the runbooks we need to audit the documents for sensitive information (credentials, internal URLs, customer-specific data)
+   - Create sanitized versions removing or abstracting sensitive content, moving the sensitive information to the customers' repositories.
+   - Establish guidelines for what constitutes sensitive information (this is already done)
+3. **Standardized structure**: Implement a consistent format for all runbooks including:
+   - **Title and description**
+   - **Prerequisites and required access**
+   - **Step-by-step procedures** with clear numbering
+   - **Expected outcomes and verification steps**
+   - **Troubleshooting section** (optional)
+   - **Related runbooks and references** (optional)
```

Comment:
I don't think such major rework should be part of the migration, and I'm worried that a prescribed format just won't work because of the various styles existing runbooks were written in. Suggestion: remove this step.

We could however use a structured header (as in our RFCs, see [check script](https://github.com/giantswarm/rfc/blob/main/.github/rfc-format/check-rfc-format.py)) to ensure that newly-added ones have certain information as desired below (owner, etc.).

Reply:
I have changed to a standardized metadata.